app/experimenter/experiments/forms.py

```diff
@@ -96,7 +97,16 @@ def save(self, *args, **kwargs):
             old_values["variants"] = None
             new_values["variants"] = self.new_serialized_vals["variants"]
 
-        if self.changed_data:
+        if experiment.archived or "archived" in self.changed_data:
```

Comment:
What happens if someone edits an archived experiment?

Reply:
Ugh sorry I was totally out to lunch when I reviewed this the first time, I was too caught in the details.